src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,330 @@
+    </div>
+
+    <highlight-code type="html">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">
+
+                <progress class="progress-bar" value="60" max="100"/>
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button
+                    class="icon-btn"
+                    type="button"
+                    aria-label="Cancel upload"
+                >
+                    <svg
+                        class="icon icon--16"
+                        height="16"
+                        width="16"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-close-16"/>
+                    </svg>
+                </button>
+            </div>
+        </div>
+    </highlight-code>
+
+    <h3>Uploaded image</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div style="height: 168px; width: 168px;">
+                <div class="file-preview-card">
+                    <div class="file-preview-card__center">
+                        <img
+                            src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                            alt="Image"
+                        >
+                    </div>
+                    <div class="file-preview-card__menu-action">
+                        <button
+                            class="icon-btn"
+                            type="button"
+                            aria-label="Open menu"
+                        >
+                            <svg
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-overflow-vertical-16"/>
+                            </svg>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <highlight-code type="html">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">
+                <img
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                    alt="Image"
+                >
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg
+                        class="icon icon--16"
+                        height="16"
+                        width="16"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-overflow-vertical-16"/>
+                    </svg>
+                </button>
+            </div>
+        </div>
+    </highlight-code>
+
+    <h3>Uploaded video</h3>
+    <div class="demo">
```

Comment:
This example is the first to introduce a new element of interest (currently named `file-preview-card__label`) so it would be worth a short sentence explaining what its purpose is (seems like a preview action button?)